src/page/index.ts

```diff
@@ -0,0 +1,10 @@
+export { MenuContentPage } from './menu-content.page';
+export { ProductAddedModalPage } from './product-added-modal.page';
+export { AddressStep } from './address-step.page';
```

Comment:
add page at the end. For example: `export { AddressStepPage } from './address-step.page';`